src/Tools/MicroBuild/Build.proj

```diff
@@ -31,7 +31,7 @@
     <MSBuild Projects="$(ProjectDir)src\NuGet\NuGet.proj" />
     <MSBuild Projects="$(ProjectDir)src\Setup\SetupStep2.proj" />
 
-    <MSBuild Projects="$(ProjectDir)BuildAndTest.proj" Targets="Test" />
+    <MSBuild Projects="$(ProjectDir)BuildAndTest.proj" Targets="Test" Condition="'$(SkipTest)' == ''" />
```

Comment:
They pass just fine in builds.  I just added this parameter so we could avoid them during CI validation.  Wanted to make this job about validating the build only.  Test are already run many times elsewhere.